modulo3/projeto-labefy/projeto-labefy/src/components/AddPlayList.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import axios from 'axios';
+
+
+
+class AddPlayList extends React.Component {
+  state = {
+    playListInput:"",
+  }
+
+  handlePlayListInput = (event) => {
+    this.setState({playListInput:event.target.value})
+ }
+
+ creatPlayList = async() =>{
+   const url = 'https://us-central1-labenu-apis.cloudfunctions.net/labefy/playlists'
+ 
+   const body = {
+     playlist: this.state.playListInput
```

Comment:
A chave deveria ser name, ou seja, ficaria name: this.state.playListInput. Dessa forma funcionaria a criação de playlists